app/assets/stylesheets/characters.scss

```diff
@@ -0,0 +1,19 @@
+// Place all the styles related to the characters controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
+
+.row {
+	padding-top: 20px;
+}
+.char-col {
+	padding-bottom: 20px;
+}
+
+.char-header {
+	color: #C09448;
+	padding-top: 20px;
+}
+
+.space {
```

Comment:
I don't feel like `.space` is clear enough is a space for what? or is it left, bottom, top....